x-pack/plugins/security_solution/common/endpoint/schema/resolver.ts

```diff
@@ -33,6 +33,11 @@ export const validateEvents = {
     afterEvent: schema.maybe(schema.string()),
     legacyEndpointID: schema.maybe(schema.string({ minLength: 1 })),
   }),
+  body: schema.nullable(
```

Comment:
I tried using `schema.maybe()` here but kibana returns a 400 if body does exist in the request. `nullable` works though 🤷